4-projects/README.md

```diff
@@ -211,6 +218,14 @@ grep -rl 10.3.64.0 business_unit_2/ | xargs sed -i 's/10.3.64.0/10.4.64.0/g'
    git push origin development
    ```
 
+1. Run `terraform init` in the `development` folder to generate the outputs required by step `5-app-infra`.
+
+```bash
+cd business_unit_1/development
+terraform init
+cd ../..
+```
```

Comment:
se esse codigo for ser usado para pegar um output gerado pelo cloud build vc tem que esperar ele terminar de rodar para rodar esse codigo.
Isso não pode ser pego pelo remote state?